ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt

```diff
@@ -0,0 +1,22 @@
+Ballerina Shell Help
+
+Type a ballerina statement, expression or a declaration.
+
+Imports should be done with prefixes and the modules
+should be available in the classpath.
```

Comment:
This text is not so user friendly. What are we trying to ask the user?

Reply:
This message is outdated. I'll remove this section.